### DISCUSSION TITLE
Coupling Matrix

### DISCUSSION THREAD

**Discussion by elafmusa** [atcollab/at#486] [Python AT]
I am working on implementing coupling correction on PyAT using LOCO, i used the 4X4 coupling matrix parameter of linopt function,
My question is, if i want to know the effectiveness of the correction that i implement by comparing the coupling before and after the correction, how can i do this comparison using the multiple 4X4 matrices along the Lattice ?

**Comment by simoneliuzzo** [comment]
dear @elafmusa ,

usually I simply look at the vertical emittance. If you find out I am happy to know as well!
